src/frontend/components/leftNav.jsx

```diff
@@ -0,0 +1,39 @@
+import { AiFillHome } from "react-icons/ai";
+import { FiHash } from "react-icons/fi";
+import { IoMdNotifications } from "react-icons/io";
+import { BiMessageSquareDetail } from "react-icons/bi";
+import { BsFillBookmarkFill } from "react-icons/bs";
+import { FaListAlt } from "react-icons/fa";
+import { MdAccountCircle } from "react-icons/md";
+import { CgMoreO, CgMore } from "react-icons/cg";
+
+const LeftNav = () => {
+    return (
+        <>
+            <nav className="flex flex-col min-w-2/6 h-screen items-start px-14 py-10 justify-between">
+                <ul className="flex flex-col items-start min-w-full text-xl font-semibold gap-4">
+                    <img src="assets/spectrum.png" alt="logo" className="w-14 h-14 rounded-full mb-10 shadow-[4.0px_8.0px_8.0px_rgba(0,0,0,0.38)] cursor-pointer" />
+                    <li className="flex items-center gap-5 cursor-pointer"><AiFillHome />Home</li>
+                    <li className="flex items-center gap-5 cursor-pointer"><FiHash /><p>Explore</p></li>
+                    <li className="flex items-center gap-5 cursor-pointer"><IoMdNotifications/><p>Notifications</p></li>
+                    <li className="flex items-center gap-5 cursor-pointer"><BiMessageSquareDetail /><p>Messages</p></li>
+                    <li className="flex items-center gap-5 cursor-pointer"><BsFillBookmarkFill /><p>Bookmarks</p></li>
+                    <li className="flex items-center gap-5 cursor-pointer"><FaListAlt /><p>Lists</p></li>
+                    <li className="flex items-center gap-5 cursor-pointer"><MdAccountCircle /><p>Profile</p></li>
+                    <li className="flex items-center gap-5 cursor-pointer"><CgMoreO /><p>More</p></li>
```

Comment:
list can be looped instead of repetition.
you can store you values in a constant file and use that here to loop over.